python/cucim/src/cucim/skimage/_vendored/_ndimage_filters_core.py

```diff
@@ -1,24 +1,184 @@
 """A vendored subset of cupyx.scipy.ndimage._filters_core"""
+import warnings
 
 import cupy
 import numpy
 
+try:
+    from cupy_backends.cuda.api import runtime
+    _is_hip = runtime.is_hip
+except (ImportError, AttributeError):
+    _is_hip = False
```

Comment:
Is there a reason this is needed or could we drop it?

Reply:
It was present upstream in CuPy, but it shouldn't hurt to remove it here.